dev/tests/functional/tests/MFTF/DevDocs/Test/DeprecatedDevDocsTest.xml

```diff
@@ -22,6 +22,8 @@
         <!--   Open MFTF DevDocs Page -->
         <amOnPage stepKey="openMFTFDevDocPage" url="{{DeprecatedMFTFDocPage.url}}" />
         <see stepKey="verifyPageIntroText" selector="{{DeprecatedContentSection.pageIntro}}" userInput="{{DeprecatedMessageData.message}}" />
-        <actionGroup ref="DeprecatedCommentActionGroup" stepKey="commentActionGroup" />
+        <actionGroup ref="DeprecatedCommentActionGroup" stepKey="commentActionGroup">
+            <argument name="test" value="{{DeprecatedContentSection.pageIntro}}" />
+        </actionGroup>
```

Comment:
How about define this in a new action group instead of adding in the "deprecated" test? like HelperInActionGroupTest or something?

Reply:
Done